documentation.html

```diff
@@ -124,7 +125,9 @@ <h3>Example of Alert</h3>
           </div>
         </div>
         <!-- Alert Ends -->
+
         <hr />
+
         <!-- Avatar Starts -->
         <div id="avatar">
```

Comment:
instead of using div you can use the section here it gives better clarity of your component.

Reply:
I have used section as the outer container and inside it for every component I am using div